DigitalLearningSolutions.Data.Tests/Services/CourseAdminFieldsServiceTests.cs

```diff
@@ -74,21 +78,103 @@ public void GetCustomPromptsWithAnswersForCourse_Returns_Populated_List_of_Custo
         public void UpdateCustomPromptForCourse_calls_data_service()
         {
             // Given
-            A.CallTo(() => courseAdminFieldsDataService.UpdateCustomPromptForCourse(1, 1, true, null)).DoesNothing();
+            A.CallTo(() => courseAdminFieldsDataService.UpdateCustomPromptForCourse(1, 1, null)).DoesNothing();
 
             // When
-            courseAdminFieldsService.UpdateCustomPromptForCourse(1, 1, true, null);
+            courseAdminFieldsService.UpdateCustomPromptForCourse(1, 1, null);
 
             // Then
-            A.CallTo(() => courseAdminFieldsDataService.UpdateCustomPromptForCourse(1, 1, true, null))
-                .MustHaveHappened();
+            A.CallTo(() => courseAdminFieldsDataService.UpdateCustomPromptForCourse(1, 1, null)).MustHaveHappened();
+        }
+
+        [Test]
+        public void GetCoursePromptsAlphabeticalList_calls_data_service()
+        {
+            // Given
+            const string promptName = "Access Permissions";
+            A.CallTo(() => courseAdminFieldsDataService.GetCoursePromptsAlphabetical()).Returns
+                (new List<(int, string)> { (1, promptName) });
+            // When
+            var result = courseAdminFieldsService.GetCoursePromptsAlphabeticalList();
+
+            // Then
+            A.CallTo(() => courseAdminFieldsDataService.GetCoursePromptsAlphabetical()).MustHaveHappened();
+            result.Contains((1, promptName)).Should().BeTrue();

```

Comment:
Can we just check that the result is what we return? Also, minor, but missing empty line before //when